rising/transforms/functional/painting.py

```diff
@@ -0,0 +1,118 @@
+import torch
+
+__all__ = ["local_pixel_shuffle", "random_inpainting", "random_outpainting"]
+
+
+def local_pixel_shuffle(data: torch.Tensor, n: int = -1, block_size: tuple = (0, 0, 0), rel_block_size: float = 0.1) -> torch.Tensor:
+
+    batch_size, channels, img_rows, img_cols, img_deps = data.size()
+
+    if n < 0:
+        n = int(1000 * channels)  # changes ~ 12.5% of voxels
+    for b in range(batch_size):
+        for _ in range(n):
+            c = torch.randint(0, channels - 1, (1,))
```

Comment:
```suggestion
            c = torch.randint(0, max(1, channels - 1), (1,))
```
If single channel/modality input (e.g. T2 only) was used this errored, since max=min=0 and torch.randint requires max> min.